python/couch-app.py

```diff
@@ -67,10 +67,10 @@ def index():
     return app.send_static_file('index.html')
 
 # Serve up static resources, JavaScript, HTML, CSS
-@app.route('/<path:path>')
-def js_proxy(path):
+#@app.route('/<path:path>')
```

Comment:
correct - I commented those lines out, so they can be activated again once we agree on the overall structure. Until then they would be a security issue